objectscale-manager/templates/objectscale-manager-app-configmap.yaml

```
@@ -27,6 +27,16 @@ data:
           args:
             - -target-version
             - {{ default .Values.tag .Values.healthChecks.preUpdate.image.tag}}
+        - name: cloudiq-inventory-report
+          container: {{.Values.global.registry}}/objectstore-inventory:{{.Values.tag}}
```

should be objectscale right?  since its getting inventory for all objectstores.

```suggestion
          container: {{.Values.global.registry}}/objectscale-inventory:{{.Values.tag}}
```

---

Fixed